examples/style_sheets/style_sheets_reference.py

```diff
@@ -63,7 +63,8 @@ def plot_colored_circles(ax, prng, nb_samples=15):
     the color cycle, because different styles may have different numbers
     of colors.
     """
-    for sty_dict, j in zip(plt.rcParams['axes.prop_cycle'], range(nb_samples)):
```

Comment:
I think we need to remove the comment above now?

Reply:
actually, on further reading I think the comment above is now more correct (with the `range(nb_samples)` always being exhausted now rather than sometimes cutting short based on the number of colors.